src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -87,26 +79,31 @@ const ProjectExternalSubmissionsTable = ({
         render: (s: ExternalFormSubmissionFieldsFragment) =>
           parseAndFormatDateTime(s.submittedAt),
       },
-      ...customColumns,
     ];
   }, []);
 
   const [selected, setSelected] =
     useState<ExternalFormSubmissionFieldsFragment | null>(null);
 
+  const { data: { externalFormDefinition: definition } = {}, error } =
+    useGetExternalFormDefinitionQuery({
+      variables: {
+        formDefinitionIdentifier: formDefinitionIdentifier,
+        id: selected?.definition?.id,
```

Comment:
Let's discuss whether it makes sense to use an approach more similar to assessments, where the definition to use for viewing the assessment is loaded on the assessment object itself:
```
externalFormSubmission(id: 3) {
    definition {... } // form_submission.form_processer.definition_id reference
    values {... }
}
```